packages/eslint-config/shared.js

```diff
@@ -151,25 +151,16 @@ module.exports = {
       parser: '@typescript-eslint/parser',
       extends: ['plugin:@typescript-eslint/recommended', 'prettier'],
       rules: {
-        // 'recommended' rules we alter
-        '@typescript-eslint/no-explicit-any': 'warn',
+        // TODO: look into enabling these eventually
+        '@typescript-eslint/no-empty-function': 'off',
```

Comment:
I tried turning this on, but didn't see any errors being reported.
Same for the rule below.

Am I doing something wrong? Or was this to limit the effect of this PR on user's apps?

Reply:
Oh shoot sorry @Tobbe, I missed this comment. It was to limit the effect of this PR on user's apps. I remember that _some_ of these had >0 failures but didn't pay attention to which.